src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -365,7 +369,18 @@
        :value            input-amount
        :on-swap          swap-between-fiat-and-crypto
        :on-token-press   show-select-asset-sheet
-       :allow-selection? false}]
+       :allow-selection? false
+       :crypto?          crypto-currency?
+       :converted-value  (if crypto-currency?
+                           (utils/prettify-balance
+                            currency-symbol
+                            (money/crypto->fiat input-amount
+                                                conversion-rate))
+                           (utils/prettify-crypto-balance
+                            (or (name token-symbol) "")
```

Comment:
Be careful with this piece of code, if `token-symbol` isn't comming (is `nil`) we'll get an exception.

Seems this code is expecting `token-symbol` to be `nil` sometimes, but:

![image](https://github.com/user-attachments/assets/ddd35b33-6bc3-41a7-89bd-a2356c8b5411)

we could use `(some-> token-symbol name)` instead to avoid the exception. wdyt?

Reply:
Thanks for spotting, In other parts of code `(clj->js)` used because gives same result when applied to strings or keywords, but do not throw on `nil`